src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
 
+import { PostInfo } from '../PostInfo';
+
+import postsFromServer from '../../api/posts';
+
 export const PostList: React.FC = () => (
-  <>Put the list here</>
+  <div className="PostList">
+    {postsFromServer
```

Comment:
where are you getting these `postsFromServer` from 
you should take` posts` array as a prop
to map , you should map from `posts`
`posts.map`